front-end/src/pages/player/player.tsx

```diff
@@ -129,6 +130,8 @@ function PlayerPage()
       setRoleId,
     }
   };
+  let shouldShowStatsPage = process.env.NODE_ENV === 'development';
```

Comment:
Maybe lets factor that out into an actual utitlity function. Cuz I imagine this will be used more in the future.

Reply:
Will do in my next PR

Reply:
Cringe. My team would NEVER let this go through!!!